doc/source/data/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -451,14 +478,15 @@
    },
    "outputs": [],
    "source": [
-    "prediction_batch = predictions.take_batch(5)"
+    "prediction_batch = predictions.take_batch(5)\n"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We see that all the images are correctly classified as \"tench\", which is a type of fish."
+    "We see that most of the images are correctly classified!"
```

Comment:
Are we no longer getting correct results?

Reply:
Actually looks like all 5 images are correctly classified.

Btw, I think would be good keep the clarification that tench is fish. Don't think most people know that

Reply:
Oh, got it. I changed this on the other copy because the displayed images are actually different from these ones, and the results are not all correct on that one. Plus the order is not preserved, so the output may changed from being the tench in the future if something under the hood changes?

Happy to change this for the one that's shown on OSS docs though.